setup.py

```diff
@@ -37,7 +37,7 @@
     # TODO(brendanr): Make these point to released versions.
     install_requirements = [
         "allennlp @ git+ssh://git@github.com/allenai/allennlp@master#egg=allennlp",
```

Comment:
Not a pip version here?  I would have thought we'd want a pin like `allennlp>=0.9`.

Reply:
Oh, I see the TODO above.  Still not sure why `allennlp` can't be a released version now.  `semparse` can't, of course, because there are no releases.